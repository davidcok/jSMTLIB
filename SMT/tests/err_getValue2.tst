; get-value without check-sat
(set-option :produce-models true)
(set-logic QF_UF)
(declare-fun x () Bool)
(get-value (x))
