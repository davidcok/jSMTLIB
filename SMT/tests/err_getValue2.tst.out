unsupported
success
success
(get-value (x))
^^^^^^^^^^^^^^^
(error "The get-value command is only valid if :produce-models has been enabled")
